src/controls.py

```diff
@@ -6,35 +6,49 @@
 import pygame
 
 
-class ControlType(Enum):
-    key = "key"
-    mouse = "mouse"
-
-
 @dataclass
 class Control:
     """
-    Instances of this dataclass refer to an individual Control / Keybind and its respective type, text, value and state
+    Instances of this dataclass refer to an individual Control / Keybind and
+    its respective type, text, value and state.
+    When creating a new Control object, you should only initialise
```

Comment:
You didn't finish your sentence here...

Reply:
Oh, I didn't notice that 😅 
I initially wanted to indicate that only control_value and text should be set on initialisation, but then I read about the option to just add `init=None` to each dataclass field that shouldn't be manually initialised. After that I seem to have forgotten that I already started writing that sentence, I think it can be deleted.